redpipe-engine/src/main/java/net/redpipe/engine/swagger/RxModelConverter.java

```diff
@@ -0,0 +1,52 @@
+package net.redpipe.engine.swagger;
+
+import io.reactivex.Observable;
+import io.reactivex.Single;
+import io.swagger.converter.ModelConverter;
+import io.swagger.converter.ModelConverterContext;
+import io.swagger.models.Model;
+import io.swagger.models.properties.Property;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.ParameterizedType;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class RxModelConverter implements ModelConverter {
+
+    private static final List<String> delegateToFirstTypeArg = new ArrayList<>();
+    static {
+        delegateToFirstTypeArg.add(Single.class.getName());
```

Comment:
Not sure between `io.reactivex.*` and `rx.*` so I handled both equally

Reply:
Yes, both should be supported indeed.